### DISCUSSION TITLE
what does wavy underline mean when there is no diagnostics?

### DISCUSSION THREAD

**Discussion by ssh352** [mrcjkb/rustaceanvim#390] [Q&A]
<img width="2548" alt="Screenshot 2024-05-14 at 11 56 09 PM" src="https://github.com/mrcjkb/rustaceanvim/assets/41779116/14002e25-5ed2-4ddc-b224-d0fd0fc9301a">

This is when I open library/alloc/src/str.rs in rust source tree, with lazyvim.
Please advise, thanks!

**Comment by mrcjkb** [comment]
No idea 🤷 

LazyVim comes with a lot of plugins and custom configurations, so you're probably better off asking there. 

Possibly related: https://github.com/mrcjkb/rustaceanvim/discussions/311

**Reply by ssh352** [reply]
Thanks, I'll ask there.
